samples/graph-activity-feed/csharp/ActivityFeedNotification/Model/TaskModuleInfo.cs

```diff
@@ -1,11 +1,9 @@
 ﻿using System;
-using System.Collections.Generic;
-using Microsoft.AspNetCore.Mvc.Rendering;
 
 namespace TabActivityFeed.Model
 {
     
-    public class TaskInfo
+    public class TaskModuleInfo
```

Comment:
Is it possible to rename it this something unrelated to features in the Teams SDK. 

Task Modules and Task Info are names that might cause confusion.

Something like `TaskDetails` might be better

Reply:
Yes, changed it to TaskDetails